var/spack/repos/builtin/packages/fastjet/package.py

```diff
@@ -114,6 +114,11 @@ class Fastjet(AutotoolsPackage):
         when="@3.4:3.4.1 +atlas",
         level=0,
     )
+    patch(
+        "https://gitlab.com/fastjet/fastjet/-/merge_requests/8.diff",
+        sha256="acc00a619c449552732935ec7a3bf8b191692bfff507cd01a0130c2b9586ba32",
+        level=0,
```

Comment:
```suggestion
        when="@:3.4.3",
        level=0,
```
and maybe there's a version below which this does not apply?

Reply:
As far as I can tell, this class has looked this way for 17 years. I'm not sure which lower version to put here. I can experimentally try to test this if you think it's necessary.

Added this, and also increased the `level` to 1, which was needed for the `.diff` version to apply correctly.

Reply:
Seems the plugin has been there since 2.4.0.